client/src/app/api/keys/route.ts

```diff
@@ -0,0 +1,16 @@
+import { NextResponse } from "next/server";
+import { mapping } from "@ada-anvil/metadraft-validator";
+
+export async function GET() {
+  try {
+    const keys = Object.keys(mapping);
```

Comment:
does this have to run server side? cant `mapping` be accessed directly? I dont understand why this is an api route lol

Reply:
nope it can't deno use a package that is only useable in server side so I have to do this work around to have the list of rules name on client side
(already talk with tommy today, for this an it was the best solution to keep de csv reader)

Reply:
if it has to run serverside, I dont think an api route is the best. I think it should be in a server action. But thats just my opinion😊 Do as tommy says, you guys got this 👍